test/new-e2e/tests/process/linux_test.go

```diff
@@ -0,0 +1,100 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+package process
+
+import (
+	"testing"
+	"time"
+
+	"github.com/DataDog/test-infra-definitions/components/datadog/agentparams"
+	"github.com/stretchr/testify/assert"
+
+	"github.com/DataDog/datadog-agent/test/fakeintake/aggregator"
+	"github.com/DataDog/datadog-agent/test/new-e2e/pkg/utils/e2e"
+)
+
+type linuxTestSuite struct {
+	e2e.Suite[e2e.FakeIntakeEnv]
+}
+
+func TestLinuxTestSuite(t *testing.T) {
+	e2e.Run(t, &linuxTestSuite{},
+		e2e.FakeIntakeStackDef(
+			e2e.WithAgentParams(agentparams.WithAgentConfig(processCheckConfigStr))))
+}
+
+func (s *linuxTestSuite) SetupSuite() {
+	s.Suite.SetupSuite()
+
+	// Start a process and keep it running
+	s.Env().VM.Execute("sudo apt-get -y install stress")
+	s.Env().VM.Execute("nohup stress -d 1 >myscript.log 2>&1 </dev/null &")
+}
+
+func (s *linuxTestSuite) TestProcessCheck() {
+	s.UpdateEnv(e2e.FakeIntakeStackDef(
+		e2e.WithAgentParams(agentparams.WithAgentConfig(processCheckConfigStr))))
```

Comment:
How is the agent version determined? Is it just latest?

Reply:
Yes, by default it uses `latest`. I will eventually follow up with a PR to add this test to CI on merges to `main`, and that will pull the pipeline version for the test.

Reply:
I ended up adding the CI changes in this PR since they were simple

Reply:
By default the CI uses the pipeline build, thanks to `E2E_PIPELINE_ID` passed in env var in the root e2e job